### PR TITLE
Fixing id set duplicates check

### PR DESCRIPTION
**Pull request by PA-Rweins**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/38085

## Description
Duplicate warnings were correct but versions check should be first. Moved it to beginning of duplicate check

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -989,7 +989,7 @@ def create_id_set(**kwargs):
     """Create the content dependency tree by ids."""
     check_configuration_file('create-id-set', kwargs)
     id_set_creator = IDSetCreator(**kwargs)
-    id_set_creator.create_id_set()
+    id_set_creator.create_id_set(fail_on_duplicates=True)
```
Comment (roysagi) [inline demisto_sdk/__main__.py:992]: make this a flag

**Review by roysagi — CHANGES_REQUESTED** [review]
@PA-Rweins - please add a changelog?

File: demisto_sdk/__main__.py (hunk 1 above)
Comment (PA-Rweins) [reply demisto_sdk/__main__.py]: Changed to a flag

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 * Added a version differences section to readme in **generate-docs** command.
 * Added the *--docs-format* flag in the **integration-diff** command to get the output in README format.
 * Added the *--input-old-version* and *--skip-breaking-changes* flags in the **generate-docs** command to get the details for the breaking section and to skip the breaking changes section.
+* Added the `-fd` flag for the **create-id-set** command which will fail the command if duplicates are found.
```
Comment (teizenman) [inline CHANGELOG.md:10]:
```suggestion
* Added the `--fail-duplicates` flag for the **create-id-set** command which will fail the command if duplicates are found.
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -160,7 +160,7 @@ jobs:
               export PYTHONPATH=".:${PYTHONPATH}"
               demisto-sdk -v
 
-              CI_COMMIT_BRANCH="master" demisto-sdk create-id-set -o ./Tests/id_set.json
+              CI_COMMIT_BRANCH="master" demisto-sdk create-id-set -o ./Tests/id_set.json -fd
```
Comment (teizenman) [inline .circleci/config.yml:163]:
```suggestion
              CI_COMMIT_BRANCH="master" demisto-sdk create-id-set -o ./Tests/id_set.json --fail-duplicates
```

**Review by roysagi — APPROVED** [review]

File: .circleci/config.yml (hunk 3 above)
Comment (PA-Rweins) [reply .circleci/config.yml]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (PA-Rweins) [reply CHANGELOG.md]: Fixed

**Review by teizenman — APPROVED** [review]

**Pull request merged**